core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowExecutionsResource.scala

```diff
@@ -28,6 +29,45 @@ object WorkflowExecutionsResource {
     executionsDao.fetchOneByEid(eId)
   }
 
+  def getExpiredResults(timeToLive: Int): List[ExecutionResultEntry] = {
+    context
+      .select(
+        WORKFLOW_EXECUTIONS.EID,
+        WORKFLOW_EXECUTIONS.RESULT
+      )
+      .from(WORKFLOW_EXECUTIONS)
+      .where(
+        WORKFLOW_EXECUTIONS.STATUS
+          .eq(3.toByte)
```

Comment:
is it easy to change this to mysql enums instead of using hardcoded numbers? Their usage started spreading. I'm a little bit worried now.

Reply:
This requires refactoring, we leave it to future work.